scripts/reset_db.sql

```diff
@@ -0,0 +1,152 @@
+--
+-- PostgreSQL database cluster dump
+--
+
+SET default_transaction_read_only = off;
+
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+
+--
+-- Drop databases
+--
+
+
+
+
+
```

Comment:
the reset script does not work, I'm probably better off removing it than committing something broken...